scripts/generate-sdk/languages/python.sh

```diff
@@ -0,0 +1,150 @@
+        GIT_REPO_ID="stackit-sdk-python"
+    fi
+
+    if [[ -z ${SDK_REPO_URL} ]]; then
+        echo "SDK_REPO_URL not specified, default will be used."
+        SDK_REPO_URL="https://github.com/stackitcloud/stackit-sdk-python.git"
+    fi
+
+    # Prepare folders
+    if [[ ! -d $SERVICES_FOLDER ]]; then
+      mkdir -p "$SERVICES_FOLDER"
+    fi
+
+    # Clone SDK repo
+    if [ -d ${SDK_REPO_LOCAL_PATH} ]; then
+        echo "Old SDK repo clone was found, it will be removed."
+        rm -rf ${SDK_REPO_LOCAL_PATH}
+    fi
+    git clone --quiet ${SDK_REPO_URL} ${SDK_REPO_LOCAL_PATH}
+
+    # Install SDK project tools
+    cd ${ROOT_DIR}
+    make project-tools
+
+    # Backup of the current state of the SDK services dir (services/)
+    sdk_services_backup_dir=$(mktemp -d)
+    if [[ ! ${sdk_services_backup_dir} || -d {sdk_services_backup_dir} ]]; then
+        echo "Unable to create temporary directory"
+        exit 1
+    fi
+    cleanup() {
+        rm -rf ${sdk_services_backup_dir}
+    }
+    cp -a "${SERVICES_FOLDER}/." ${sdk_services_backup_dir}
+
+    # Cleanup after we are done
+    trap cleanup EXIT
+
+    # Remove old contents of services dir (services/)
+    rm -rf ${SERVICES_FOLDER}    
+
+    # Generate SDK for each service
+    for service_json in ${ROOT_DIR}/oas/*.json; do
+        service="${service_json##*/}"
+        service="${service%.json}"
+
+        # Remove invalid characters to ensure a valid Go pkg name
+        service="${service//-/}"                                  # remove dashes
+        service="${service// /}"                                  # remove empty spaces
+        service="${service//_/}"                                  # remove underscores
+        service=$(echo "${service}" | tr '[:upper:]' '[:lower:]') # convert upper case letters to lower case
+        service=$(echo "${service}" | tr -d -c '[:alnum:]')       # remove non-alphanumeric characters
+
+        echo "Generating \"${service}\" service..."
+        cd ${ROOT_DIR}
+
+        mkdir -p "${SERVICES_FOLDER}/${service}/"
+        cp "${ROOT_DIR}/scripts/generate-sdk/.openapi-generator-ignore" "${SERVICES_FOLDER}/${service}/"
+
+        # Run the generator
+        java -Dlog.level=${GENERATOR_LOG_LEVEL} -jar ${jar_path} generate \
+            --generator-name python \
+            --input-spec "${service_json}" \
+            --output "${SERVICES_FOLDER}/${service}" \
+            --package-name "stackit.${service}" \
+            --template-dir "${ROOT_DIR}/templates/python/" \
+            --git-host ${GIT_HOST} \
+            --git-user-id ${GIT_USER_ID} \
+            --git-repo-id ${GIT_REPO_ID} \
+            --global-property apis,models,modelTests=false,modelDocs=false,apiDocs=false,apiTests=false,supportingFiles \
+            --additional-properties=pythonPackageName="stackit-${service},removeEnumValuePrefix=false" >/dev/null
+        ##--enable-post-process-file \
```

Comment:
remove the line instead of commenting out

Reply:
done